cmd/storage/main.go

```diff
@@ -228,3 +274,15 @@ func run(db *sqlx.DB, errc chan<- error) {
 	log.Info("serve Swagger REST protocol", def.LogHost, cfg.extapi.Host, def.LogPort, cfg.extapi.Port)
 	errc <- extsrv.Serve()
 }
+
+func checkTime(chck bool) {
```

Comment:
Let's name it as `waitUntilSysTimeIsCorrect` or something. Quiet long name, but much more meaningful.